### ISSUE TITLE
Solanian's Belongings - Scrying Orb

### ISSUE THREAD

**Issue by Blooddrainer45** [dalaranwow/dalaran-wow#5165]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: When clicking upon Solanian's Scrying Orb to loot for the quest, there is no quest item.

**Expected behaviour**: When clicking upon Solanian's Scrying Orb to loot for the quest, you're meant to obtain the quest item.

**Steps to reproduce the problem**:

1. Loot the quest item.
2. Obtain the quest item.
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**[quest=8330] [item=20470]**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello @Blooddrainer45,
I've tested this issue on PTR and I can't confirm it, for me the loot is there.
![image](https://user-images.githubusercontent.com/6797611/33994141-12fa2aa8-e0e2-11e7-91b5-9dd609533bdd.png)

**Comment by Darkshape45** [comment]
I just attempted to loot Solanian's Scrying Orb and it was not present in in the quest object.  I used shift + right click to auto-pickup the loot but no box opened and I got nothing.  I then used just right click to check inside and it was empty.  I dropped the quest and picked it up again, but that did not allow the object to appear for looting.

**Comment by MarcusTherion** [comment]
Can confirm that this has also just happened to me, quest present, both other items looted, but right clicking the orb did nothing, shift right clicking just opened the loot window with no orb to loot. 

![image](https://user-images.githubusercontent.com/39283382/40027816-afedb8b2-57d3-11e8-8ba4-41e3a606a205.png)
